lib/cli/scripts/licenses.ts

```diff
@@ -148,9 +152,19 @@ export default function main(_args: string[], workingDir: string) {
                     console.error(err);
                     reject(err);
                 } else {
+                    const filteredPackages = Object.keys(directDependencies).reduce((obj, packageName) => {
+                        obj[packageName] = packages[packageName];
+                        return obj;
+                    }, {});
+
+                    // eslint-disable-next-line no-console
+                    console.log({ filteredPackages });
```

Comment:
Do we need this console log and that one below?

Reply:
removed